revolv/templates/project/project.html

```diff
@@ -91,4 +93,127 @@
     }
     google.maps.event.addDomListener(window, 'load', initialize);
 </script>
+
+<script type="text/javascript">
+$(function() {
```

Comment:
Aish this is a lot of javascript, you were right. I guess it's necessary though. Can you at least use this script tag to populate variables that you need from the template, then do all the processing in a separate js file?